### PR TITLE
[fix](bloom filter)Fix rename column with bloom filter

### LINKED ISSUE #38643: [Bug] rename bloom filter column succeeded, but the bloom_filter_columns in properties is not updated.
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/doris/issues?q=is%3Aissue) and found no similar issues.

### Version

master

### What's Wrong?

rename bloom filter column succeeded, but the bloom_filter_columns in properties is not updated.

### What You Expected?

rename bloom filter column succeeded,and the bloom_filter_columns in properties is updated too.

### How to Reproduce?

1. create table
```
create table type_test_mor (
    k1 INT,
    v VARIANT,
    v2 VARCHAR(300),
    s STRING,
    INDEX v (`v`) USING INVERTED,
    INDEX v2(`v2`) USING BITMAP,
    INDEX s(`s`) USING NGRAM_BF PROPERTIES("gram_size"="3", "bf_size"="256")
) unique key(k1) distributed by hash(k1) buckets 1
properties(
    "bloom_filter_columns" = "k1",
    "replication_num" = "1",
    "enable_unique_key_merge_on_write" = "false"
);
```
2. rename bloom filter column
```
alter table type_test_mor rename column k1 k1_renamed;
```
3. show create table
```
CREATE TABLE `type_test_mor` (
  `k1_renamed` INT NULL,
  `v` VARIANT NULL,
  `v2` VARCHAR(300) NULL,
  `s_renamed` TEXT NULL,
  INDEX v (`v`) USING INVERTED,
  INDEX v2 (`v2`) USING INVERTED,
  INDEX s (`s`) USING NGRAM_BF PROPERTIES("bf_size" = "256", "gram_size" = "3")
) ENGINE=OLAP
UNIQUE KEY(`k1_renamed`)
DISTRIBUTED BY HASH(`k1_renamed`) BUCKETS 1
PROPERTIES (
"replication_allocation" = "tag.location.default: 1",
"min_load_replica_num" = "-1",
"bloom_filter_columns" = "k1",
"is_being_synced" = "false",
"storage_medium" = "hdd",
"storage_format" = "V2",
"inverted_index_storage_format" = "V2",
"enable_unique_key_merge_on_write" = "false",
"light_schema_change" = "true",
"disable_auto_compaction" = "false",
"enable_single_replica_compaction" = "false",
"group_commit_interval_ms" = "10000",
"group_commit_data_bytes" = "134217728"
);
```

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by qidaye**
## Proposed changes

Issue Number: close #38643

### CODE REVIEW

**Comment by qidaye** [comment]
run buildall

**Review by xiaokang — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java [hunk 1]
```diff
@@ -5051,6 +5052,18 @@ private void renameColumn(Database db, OlapTable table, String colName,
             table.setSequenceMapCol(newColName);
         }
 
+        // 6. modify bloom filter col
+        Set<String> bfCols = table.getCopiedBfColumns();
+        Set<String> newBfCols = new HashSet<>();
+        for (String bfCol : bfCols) {
```
Comment (xiaokang) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java:5058]: You should check bfCols != null first

Comment (qidaye) [reply fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java]: done

**Comment by qidaye** [comment]
run buildall

**Comment by airborne12** [comment]
need to fill Proposed changes

**Review by xiaokang — APPROVED** [review]
LGTM

**Comment by qidaye** [comment]
> need to fill Proposed changes

Added in issue.

**Review by airborne12 — APPROVED** [review]
LGTM

**Review by hello-stephen — APPROVED** [review]

**Review by CalvinKirs — APPROVED** [review]

**Pull request merged**